Add CircleCI / Gitlab integration examples (#275)
saucelabs/sauce-docs: docs/testrunner-toolkit/integrations/circleci.md

## Modify the CirceCI Configuration

In the root of your project directory, create the `.circleci` directory if it doesn't already exist, and open/create `config.yml`. 

> Saucectl can be easily executed on CircleCI throught [saucectl-run](https://circleci.com/developer/orbs/orb/saucelabs/saucectl-run) orb. You need to add the orb to you current `config.yml` to be able to use it.

Reviewer: ```suggestion
You can easily execute Saucectl on CircleCI through [saucectl-run](https://circleci.com/developer/orbs/orb/saucelabs/saucectl-run) orb. Add the orb to your current `config.yml` using the following reference to  use it:
```

> The orb need to be referenced this way:

Reviewer: ```suggestion
```

```yaml reference
https://github.com/saucelabs/saucectl-cypress-example/blob/master/.circleci/config.yml#L2-L3
```

> Now, you can use the orb directly as a step of one of your jobs:

Reviewer: ```suggestion
Now you can use the orb directly as a step of one of your jobs:
```

```yaml reference
https://github.com/saucelabs/saucectl-cypress-example/blob/master/.circleci/config.yml#L13-L15
```

> > You can reference the complete `config.yml` [here](https://github.com/saucelabs/saucectl-cypress-example/blob/master/.circleci/config.yml).

Reviewer: ```suggestion
> You can reference a complete [`config.yml` example](https://github.com/saucelabs/saucectl-cypress-example/blob/master/.circleci/config.yml) for more detail.
```
